houzz.com.website-services.json

```diff
@@ -0,0 +1,34 @@
+{
+	"providerId":"Houzz.com",
```

Comment:
Might be better making the providerId houuzz.com; it will avoid headaches with some implementations